subprojects/testing-junit-platform/src/main/java/org/gradle/api/internal/tasks/testing/junitplatform/JUnitPlatformTestExecutionListener.java

```diff
@@ -134,17 +134,24 @@ private boolean isFailedContainer(TestIdentifier testIdentifier, TestExecutionRe
 
     private TestDescriptorInternal getDescriptor(final TestIdentifier test) {
         if (isMethod(test)) {
-            return new DefaultTestDescriptor(idGenerator.generateId(), className(test), test.getDisplayName());
+            return createDescriptor(test, test.getLegacyReportingName(), test.getDisplayName());
```

Comment:
⭕️ We changed to legacy reporting name here - can you point me to the test coverage that comes with this change?

Reply:
Corresponding tests are mostly here: https://github.com/gradle/gradle/blob/4d1df664160857a4057d83636ac0f108c4640a3e/subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junitplatform/JUnitPlatformUserGuideIntegrationTest.groovy#L28